src/core/lib/surface/channel.h

```diff
@@ -59,22 +59,76 @@ grpc_core::channelz::ChannelNode* grpc_channel_get_channelz_node(
     status_code.
 
     The returned elem is owned by the caller. */
-grpc_mdelem grpc_channel_get_reffed_status_elem(grpc_channel* channel,
-                                                int status_code);
+grpc_mdelem grpc_channel_get_reffed_status_elem_slowpath(grpc_channel* channel,
+                                                         int status_code);
+inline grpc_mdelem grpc_channel_get_reffed_status_elem(grpc_channel* channel,
```

Comment:
Do you have numbers on how much the binary size increased after the inlining?

Reply:
Not yet, but I don't expect it to change for this specific method since there's only one caller.

Reply:
That's a great question. Seems to be negligible:

```  -0.0%    -640 TOTAL                                                                                +23.3Ki  +0.1%```